product_docs/docs/pgd/5/data_migration/edbloader.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: EDB*Loader and PGD
+navTitle: EDB*Loader
+description: EDB*Loader is a high-speed data loading utility for EDB Postgres Advanced Server.
+---
+
+[EDB\*Loader](/epas/latest/database_administration/02_edb_loader/) is a high-speed data loading utility for EDB Postgres Advanced Server (EPAS). It provides an interface compatible with Oracle databases, allowing you to load data into EDB Postgres Advanced Server. It's designed to load large volumes of data into EDB Postgres Advanced Server quickly and efficiently.
+
+The EDB\*Loader command line utility loads data from an input source, which can be a file flat file, pipe, or other programs, into one or more tables using a subset of the parameters offered by Oracle SQL\*Loader. 
+
+## Use with PGD
+ 
+As EDB\*Loader is a utility for EDB Postgres Advanced Server, it's available for EDB Postgres Distributed when EDB Postgres Advanced Server is the database in use for PGD data nodes. PGD deployments can use EDB\*Loader in the same way as it's used on EDB Postgres Advanced Server. Refer to the [EDB\*Loader documentation](/epas/latest/database_administration/02_edb_loader/) for more information on how to use EDB\*Loader with EDB Postgres Advanced Server.
+
+### Replication and EDB\*Loader
+
+As with EDB Postgres Advanced Server, EDB\*Loader works with PGD in a replication environment. But, like EPAS, when using the [direct path load method](/epas/latest/database_administration/02_edb_loader/invoking_edb_loader/direct_path_load/), the replication features aren't available. Only the node connected to by EDB\*Loader gets the data that's EDB\*Loader is loading. 
+
+Unlike EPAS, which isn't able to replicate the data that's loaded using the direct path load method, PGD can replicate the data that's loaded using EDB\*Loader. This is because PGD replicates the data at the logical level, not the physical level. With PGD it's possible to run the direct load path method to one node at a time or in parallel to all nodes, depending on the use case. When doing this, it's important to ensure there are no other writes happening to the table concurrently as this may result in inconsistencies.
```

Comment:
As Adam mentioned in the original google doc, it's better to say something like "Unlike EPAS with physical replication" because physical replication is the limitation, PGD still works on EPAS so EPAS is not differentiator